jts-test-runner/src/input.rs

```diff
@@ -29,10 +30,19 @@ use super::Result;
 /// ```
 #[derive(Debug, Deserialize)]
 pub(crate) struct Run {
+    #[serde(rename = "precisionModel", default)]
+    pub precision_model: Option<PrecisionModel>,
+
     #[serde(rename = "case")]
     pub cases: Vec<Case>,
 }
 
+#[derive(Debug, Deserialize)]
```

Comment:
This tries to parse: `<precisionModel scale="1.0" offsetx="0.0" offsety="0.0"/>` as `{ ty: "" }`
and `<precisionModel type='FLOATING'/>` as `{ty: "FLOATING"}`